### PR TITLE
Fix amplicon pooling (issue #204)

### PR DESCRIPTION
**Pull request by tanaes**
This PR fixes issue #203, consolidating how amplicon and shotgun library plates are pooled. Changes in functionality:

- both shotgun and amplicon libraries can be pooled using either 'equal volume' or 'minimum volume' pooling
- for both library types, blanks can optionally be handled differently than samples, either by choosing to pool only a set number of blanks (automatically choosing the N highest-concentrated blanks), and/or by setting blank pooling values to a specific volume. Defaults are currently set differently for each library type, in line with current practices.
- estimates of the total pool concentration and volume are computed and displayed below the heatmap representation per pool
- test data now have more realistic numbers for concentration
- a number of small bug fixes in the interface

### CODE REVIEW

File: labman/gui/templates/library_pooling.html [hunk 1]
```diff
@@ -169,35 +187,34 @@
...
+        } else {
+          $.each(poolParams[plateType][poolFunc], function(idx, paramElem) {
+            var $element = $('#' + paramElem['prefix'] + plateId);
+
+            var isEmpty = $element.val() === '' || $element.val() === null;
+            var isFieldDisabled = $element.prop('disabled')
+
+            disable_compute = ((isEmpty && !isFieldDisabled) || disable_compute)
+
+            //$('#compute-pool-btn').prop('disabled', (isEmpty && !isFieldDisabled) || isCompPoolBtnDisabled);
```
Comment (ElDeveloper) [inline labman/gui/templates/library_pooling.html]: Can you remove this line?

Comment (tanaes) [reply labman/gui/templates/library_pooling.html]: Done

**Comment by tanaes** [comment]
@ElDeveloper If you're satisfied, can you merge? I want to start propagating these PRs into main.

**Comment by tanaes** [comment]
actually @ElDeveloper I think we should merge #228 first, and then update this to be compatible.

**Comment by ElDeveloper** [comment]
Should be ready for merge after tests pass.

**Comment by tanaes** [comment]
Thanks @ElDeveloper! This looks good to me. Ok to merge?

**Comment by ElDeveloper** [comment]
Good to go!

**Pull request merged**